vr4300/pipeline.c

```diff
@@ -185,6 +185,12 @@ static int vr4300_ex_stage(struct vr4300 *vr4300) {
     vr4300_opcode_mnemonics[rfex_latch->opcode.id]);
 #endif
 
+  if (vr4300->profile_samples) {
+    uint32_t idx = rfex_latch->common.pc - 0x80000000;
```

Comment:
A tiny number of these instructions can be canceled before commit as they are still in-flight - it will throw off results ever so slightly.